src/assets/stylesheets/application.scss

```diff
@@ -56,6 +56,8 @@
 @import "extensions/footnotes";
 @import "extensions/permalinks";
 
+@import "clipboardjs/clipboardjs";
```

Comment:
Create a folder called `integrations`, so the clipboard integration is located in `integrations/clipboard`.